### PR TITLE
Grammar fixes including fixing missing oxford comas.

### PR DESCRIPTION
**Pull request by rinchen**
Several grammar fixes including missing Oxford commas (Oxford comas are used throughout but there are few that were missed), correction of the use of the plural of people, and removal of an extraneous "how".

***
<a href="https://pr-preview.s3.amazonaws.com/rinchen/privacy-principles/pull/408.html" title="Last updated on Feb 28, 2024, 5:23 PM UTC (9d494f6)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/408/064c571...rinchen:9d494f6.html" title="Last updated on Feb 28, 2024, 5:23 PM UTC (9d494f6)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1046,7 +1046,7 @@
 
 Once one is choosing between different designs at the Pareto frontier, the choice of which
 privacy principles to prefer is complex and depends heavily on the details of each
-particular situation. Note that people's privacy can also be in tension
+particular situation. Note that peoples' privacy can also be in tension
```
Comment (jyasskin) [inline index.html:1049]:
```suggestion
particular situation. Note that people's privacy can also be in tension
```

File: index.html [hunk 2]
```diff
@@ -1063,7 +1063,7 @@
 This is a special case of the <a href="#no-secondary-use">more general principle that data should not be used for more
 purposes than those specified when the data was collected</a>.
 
-Services sometimes use people's data in order to protect those or other people.
+Services sometimes use peoples' data in order to protect those or other people.
```
Comment (jyasskin) [inline index.html:1066]:
```suggestion
Services sometimes use people's data in order to protect those or other people.
```

**Review by jyasskin — COMMENTED** [review]
"people's" was correct

File: index.html [hunk 3]
```diff
@@ -633,7 +633,7 @@
 
 <dfn data-lt="governance">Data governance</dfn> is the system of principles that regulate [=information flows=].
 [=Data governance=] determines how
-which [=actors=] can collect what [=data=] and how they may, must, or must not [=process=] it
+ [=actors=] can collect what [=data=] and how they may, must, or must not [=process=] it
```
Comment (darobin) [inline index.html:636]:
```suggestion
which [=actors=] can collect what [=data=] and how they may, must, or must not [=process=] it
```

That sentence isn't great but we have a better fix in.

**Review by jyasskin — APPROVED** [review]
The oxford commas look good.

**Comment by rinchen** [comment]
Thanks all.

**Pull request merged**